### ISSUE TITLE
Upload not working for packs since 1.15.0

### ISSUE THREAD

**Issue by ckaadic** [demisto/demisto-sdk#3041]
Hello,

updated to latest 1.15.0 version and get the following error when trying to upload:

```bash
(demisto) 20230522-11:11:21 wagner@DESKTOP:~/demisto/content$ demisto-sdk upload -i Packs/SIEM/ -z
You are using demisto-sdk 1.15.0.
Uploading /home/wagner/demisto/content/Packs/SIEM to https://demisto-dev.local.net...
Parsing Pack:SIEM
Dumped pack SIEM.
UPLOAD SUMMARY:

FAILED UPLOADS:
╒════════════╤════════╤════════════════════════════════════════════════════════════════════════╕
│ NAME       │ TYPE   │ ERROR                                                                  │
╞════════════╪════════╪════════════════════════════════════════════════════════════════════════╡
│ SIEM       │ Pack   │ [Errno 2] No such file or directory: '/tmp/tmpvogg2sco/SIEM.zip'       │
╘════════════╧════════╧════════════════════════════════════════════════════════════════════════╛
```

**Comment by ckaadic** [comment]
In 1.14.5 it looks like this:

```
(demisto) 20230522-11:29:32 wagner@DESKTOP:~/demisto/content$ demisto-sdk upload -i Packs/SIEM/ -z
You are using demisto-sdk 1.14.5.
however version 1.15.0 is available.
To update, run pip3 install --upgrade demisto-sdk
Using host='https://demisto-dev.local.net'
Uploading /tmp/tmpn9nwvem_/uploadable_packs.zip ...
Uploading...


UPLOAD SUMMARY:

SUCCESSFUL UPLOADS:
╒════════════╤════════╕
│ NAME       │ TYPE   │
╞════════════╪════════╡
│ SIEM       │ pack   │
╘════════════╧════════╛
```

**Comment by dorschw** [comment]
Hey, thanks
Please [contact me over Slack](https://dfircommunity.slack.com/archives/D02HQ5VQ48H)

**Comment by dorschw** [comment]
Solved in 1.15.2
